client/state/invites/test/reducer.js

```diff
@@ -0,0 +1,54 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import { requestingResend } from '../reducer';
+import { INVITE_RESEND_REQUEST } from 'state/action-types';
+
+describe( 'reducer', () => {
+	describe( '#resendRequests()', () => {
+		test( 'should key requests by site ID and invite ID', () => {
+			const state = requestingResend(
+				{},
+				{
+					type: INVITE_RESEND_REQUEST,
+					siteId: 12345,
+					inviteId: '123456asdf789',
+				}
+			);
+			expect( state ).to.eql( {
+				12345: { '123456asdf789': true },
+			} );
+		} );
```

Comment:
Super minor: I would add blank lines in between each test, for clear visual separation

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/21845.